Source/MidiHandler.cpp

```diff
@@ -117,6 +117,9 @@ void MidiHandler::handleControlChange(uint8_t status, uint8_t cc, uint8_t value)
     case SYSEX_CONFIGURATION_COMMAND:
       midi_tx.sendSettings();
       break;
+    case SYSEX_RESOURCE_NAME_COMMAND:
```

Comment:
Maybe we add size at the end of the NAME response, after the \0 filename termination?

Reply:
We could use something like SYSEX_RESOURCE_INFO_COMMAND and use a structure of fixed length name + size as its output.

If we include magic from the header there as well, it would be possible to migrate data to a different structure if that would be necessary in the future.